dev/TeachingTip/TeachingTip.cpp

```diff
@@ -830,6 +831,26 @@ void TeachingTip::OnIsOpenChanged()
                 }
             }
         }
+
+        m_acceleratorKeyActivatedRevoker = Dispatcher().AcceleratorKeyActivated(winrt::auto_revoke, { this, &TeachingTip::OnF6AcceleratorKeyClicked });
+
+        if (auto child = m_rootElement.get())
+        {
+            m_tipGettingFocusRevoker = AddRoutedEventHandler<RoutedEventType::GettingFocus>(
+                child,
+                [this](auto&& sender, auto&& args)
+                {
+                    m_hasFocusInSubtree = true;
+                },
+                true);
+            m_tipLosingFocusRevoker = AddRoutedEventHandler<RoutedEventType::LosingFocus>(
+                child,
+                [this](auto&& sender, auto&& args)
+                {
+                    m_hasFocusInSubtree = false;
```

Comment:
>m_hasFocusInSubtree [](start = 20, length = 19)

If focus moves from one element to another within the subtree of the TeachingTip, this event will still bubble up to you - so you'll set this to false incorrectly in that case.